ui/pages/setup_passkey.tsx

```diff
@@ -202,6 +290,21 @@ const SetupPasskey: NextPage = () => {
               )}
             </>
           )}
+          {isSequencedFromSignInFlow && (
+            <>
+              <hr />
+              <div className="u-align--right">
+                <Button
+                  appearance="positive"
+                  type="button"
+                  className="u-no-margin--bottom u-align--right"
+                  onClick={highlightPasskeyRequirement}
+                >
+                  Sign in
```

Comment:
Once you add a security key on the setup screen (first login), you are automatically signed in. I thought the purpose of this button is to let the user add more keys if needed and then click on `Sign in` when they're ready.

![image](https://github.com/user-attachments/assets/4c7984d0-a778-41ce-bf09-affa84270dbd)

Reply:
When we discussed it with Lukas he mentioned this button is here to let the users think they are still in the sign in flow. See my comments in the description above about the missing signal and the compromise to not allow adding multiple keys as the flow finishes automatically, and we have no way to intercept it in the frontend. If you see a way to get that signal I am happy to use it.

Reply:
It was related to my [comment](https://www.figma.com/design/zstuCMdHsKx8Wr9ERZx9Kw?node-id=630-24118#1116366810) in figma, if you'd be able to tie the `return_to` with the sign in button.
Feel free to disregard if that was agreed with Lukas.

Reply:
On adding a security key, the form is submitting to kratos with a post request (see below). The response has a redirect header to the login form. I don't see a way to intercept that. On the login page, I also don't see a way to identify the user is just adding their first key in order to send them back to the setup passkey page. Do you see a way to do that @natalian98 ?

![image](https://github.com/user-attachments/assets/cb3e19c2-1e9f-458f-b791-4cfecd1f13f8)

Reply:
I was asking about that because the settings form is sent by the webauthn script (see https://github.com/canonical/identity-platform-login-ui/blob/955e47f3b6269de9170aba91c65c8fdf28c99eb1/ui/pages/setup_passkey.tsx#L77). If you don't see a way to intercept it then let's leave it.